detailed_docs/TRS/iAP over BT/iAP_over_BT_re-connection.md

```diff
@@ -0,0 +1,77 @@
+## Functional Requirements
+
+1.
+In case device was connected over Bluetooth and SDL received signal about USB connection of the same device
+
+SDL must:
+
+- start `<AppTransportChangeTimer> + <AppTransportChangeTimerAddition>*N` after sending switch signal to external system  
+_Note:_ **N** is the number of applications that need to be re-registered
+
+2.
+In case mobile application sends RegisterAppInterface during `<AppTransportChangeTimer> + <AppTransportChangeTimerAddition>*N` timer
+and `<UUID>` and `<appID>` parameters match with closed session
+
+SDL must:
+- consider such app as re-registered app
+- update data in PolicyTable (`<device_identifier>` section)
+- send UpdataDeviceList to HMI
+
+_Note:_ For HMI during re-connection all applications remain registered and active, HMILevel resumption is not applicable
+
+3.
+In case mobile application was not re-registered during `<AppTransportChangeTimer> + <AppTransportChangeTimerAddition>*N`
+due to transport type change
+
+SDL must:
+
+send BC.OnAppUnregistered (appID, unexpectedDisconnect:true) to HMI for such application
+
+4.
+In case mobile application sends RegisterAppInterface during `<AppTransportChangeTimer> + <AppTransportChangeTimerAddition>*N` timer  
```

Comment:
@KhrystynaDubovyk , maybe better to use an 'app' instead of 'applications' as made in all rest of files?
It's for whole iAP_over_BT_re-connection.md

Reply:
Rejected.
changed 'app' into 'application' throughout  the document in favour of existing TRS documents written with 'application' (embedded_navi, RC, iAP_over_BT_data_resumptionTRS, ).